ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -429,6 +446,15 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeParcelable(v2UiCustomization, flags);
         dest.writeParcelable(v1UiCustomization, flags);
         dest.writeString(accountType);
+        if (customFields != null) {
+            dest.writeInt(customFields.size());
+            for (Map.Entry<String, Object> entry: customFields.entrySet()) {
+                dest.writeString(entry.getKey());
+                dest.writeValue(entry.getValue());
+            }
+        } else {
+            dest.writeInt(0);
```

Comment:
```suggestion
            // set custom fields size == 0 so we know not to create a customFields map when deserializing
            dest.writeInt(0);
```
Adding a comment to callout that this is related to custom fields.